examples/use_case_repository/webserver/main.py

```diff
@@ -0,0 +1,259 @@
+import os
+
+import frontmatter
+import markdown
+from dagster import file_relative_path
+from flask import Flask, abort, render_template_string
+from pygments.formatters import HtmlFormatter
+
+app = Flask(__name__)
+
+USE_CASES_DIR = file_relative_path(__file__, "../use_case_repository/guides")
+
+
+@app.route("/")
+def index():
+    use_cases = []
+    for filename in os.listdir(USE_CASES_DIR):
+        if filename.endswith(".md"):
+            with open(os.path.join(USE_CASES_DIR, filename), "r") as f:
+                post = frontmatter.load(f)
+                use_cases.append(
+                    {
+                        "title": post.get("title", "Untitled"),
+                        "description": post.get("description", ""),
+                        "tags": post.get("tags", []),
+                        "slug": filename[:-3],
+                    }
+                )
+
+    return render_template_string(
+        """
+    <!DOCTYPE html>
```

Comment:
I would like to refactor this into a separate template HTML file, but I can do that in a follow-up. 👍

Reply:
Yea this was just a simple internal dev tool that i didn't want to spend too much time on it. Got a little carried away asking AI to making it prettier